cardano-db-sync/src/Cardano/DbSync/Config/Conway.hs

```diff
@@ -36,11 +41,27 @@ readConwayGenesisConfig ::
 readConwayGenesisConfig SyncNodeConfig {..} =
   case dncConwayGenesisFile of
     Just file -> readConwayGenesisConfig' file dncConwayGenesisHash
-    Nothing -> pure (ConwayGenesis def def def mempty mempty)
+    Nothing -> pure (ConwayGenesis defaultUpgradeConwayPParams def def mempty mempty)
   where
     readConwayGenesisConfig' file hash =
       firstExceptT (SNErrConwayConfig (unGenesisFile file) . renderConwayGenesisError) $
         readGenesis file hash
+    defaultUpgradeConwayPParams :: UpgradeConwayPParams Identity
+    defaultUpgradeConwayPParams =
+      UpgradeConwayPParams
+        { ucppPoolVotingThresholds = def
+        , ucppDRepVotingThresholds = def
+        , ucppCommitteeMinSize = 0
+        , ucppCommitteeMaxTermLength = EpochInterval 0
+        , ucppGovActionLifetime = EpochInterval 0
+        , ucppGovActionDeposit = Coin 0
+        , ucppDRepDeposit = Coin 0
+        , ucppDRepActivity = EpochInterval 0
+        , ucppMinFeeRefScriptCostPerByte = minBound
+        , ucppPlutusV3CostModel = case mkCostModel PlutusV3 (replicate 233 0) of
+            Left err -> error $ "Failed to create default Plutus V3 CostModel " <> show err
```

Comment:
`logAndThrowIO` inside of `Error.hs` allows for better error handling and messaging.

Reply:
The problem is `logAndThrowIO` works in IO. This will anyway fail very early if it's wrong and it's just when Conway Genesis is ignored.

Reply:
cool that makes sense 👍